backend-tests/tests/test_azure.py

```diff
@@ -93,59 +86,25 @@ def get_settings(self, user):
         return r
 
 
-class TestAzureSettingsEnterprise(_TestAzureBase):
-    @property
-    def logger(self):
-        return logging.getLogger(self.__class__.__name__)
-
-    def test_get_and_set(self, clean_mongo):
-        """
-        Check that we can set and get settings
-        """
-        self.logger.info("creating tenant and user")
-        t = create_tenant_test_setup(
-            "azureuser0@mender.io", "t0", "somepassword10101010"
-        )
-
-        for expected_settings in [
-            {
-                "connection_string": "HostName=localhost;SharedAccessKey=thisIsBase64;SharedAccessKeyName=OldKey"
-            },
-            {
-                "connection_string": "HostName=localhost;SharedAccessKey=thisIsBase64;SharedAccessKeyName=NewKey"
-            },
-        ]:
-            r = super().save_settings(t.users[0], expected_settings)
-            assert r.status_code == 204
-            self.logger.info("saved settings")
-
-            self.logger.info("getting settings")
-            r = super().get_settings(t.users[0])
-            assert r.status_code == 200
-            self.logger.info("got settings: %s" % r.text)
-            r_json = r.json()
-            assert "connection_string" in r_json.keys()
-            actual = r_json["connection_string"]
-            # Check for equality by parts:
-            # Check that actual properties are a subset of expected settings
-            for part in actual.split(";"):
-                assert part in expected_settings["connection_string"]
-            # Check that expected properties are a subset of actual settings
-            for part in expected_settings["connection_string"].split(";"):
-                assert part in actual
-
-
 class TestAzureSettings(_TestAzureBase):
     @property
     def logger(self):
         return logging.getLogger(self.__class__.__name__)
 
-    def test_get_and_set(self, clean_mongo):
+    @pytest.mark.parametrize("plan", ["os", "enterprise"])
```

Comment:
I see what you are doing here, which is nice, but the class name needs to have `Enterprise` in it to be picked up by the right job in CI. Where applicable, we use this design of base class and os + ent child classes for this reason.

Reply:
done!